usbguard.service.in

```diff
@@ -4,8 +4,8 @@ Wants=systemd-udevd.service local-fs.target
 Documentation=man:usbguard-daemon(8)
 
 [Service]
-Type=simple
-ExecStart=%sbindir%/usbguard-daemon -k -c %sysconfdir%/usbguard/usbguard-daemon.conf
+Type=forking
```

Comment:
This change is driven by a request to improve the usability of `systemctl start usbguard` in case of a configuration file or policy file processing failure (e.g. syntax errors).

This change allows the systemctl tool to report a failure to start the service immediately. In the Type=simple case, you have to use `systemctl status usbguard` to see whether the service started and is running. The default hasn't changed for non-systemd users, as you have to specify the `-f` option to enable the forking.